style/fontsizes.less

```diff
@@ -0,0 +1,29 @@
+@standard: 13;
+@title: 21;
+@preview-title: 14;
+@desc: 11;
+
+each(range(7), {
+	.font-size-@{value} {
+		.article-content,
+		.confirm-dialog,
+		.quickfacts,
+		.preview-text,
+		.textsize-preview {
+			font-size: unit(@standard + @value, px);
+		}
+
+		.title {
+			font-size: unit(@title + @value, px);
+		}
+
+		.preview-title {
+			font-size: unit(@preview-title + @value, px);
+		}
+
+		.desc,
+		.reference-preview {
+			font-size: unit(@desc + @value, px);
+		}
+	}
+});
```

Comment:
another way of not generating these css value is using `em` for the relative font size from it's container (parent element), `<AdjustableTextSize/>` component has advantage here

```html
<div class="container">
    <AdjustableTextSize> bla bla bla </AdjustableTextSize>
</div>
```

assume container has `font-size: 10px`
default adjustableTextSize element has `font-size: 1em`

when textsize `adjust(1)` called, adjustableTextSize element change it's font-size to 1.1em; same `adjust(-1)` then `0.9em`

Reply:
so the solution is like 

**CSS** `to handle all font size value` versus **JS** `useEffect( x, dependencies) listener to change element font size em`